src/components/verification-grid/verification-grid.ts

```diff
@@ -322,6 +324,14 @@ export class VerificationGridComponent extends WithShoelace(AbstractComponent(Li
     return availableTiles - visibleSubjectCount;
   }
 
+  private get behavingSubSelection(): SubjectWrapper[] {
```

Comment:
that's a weird name?

can we make currentSubSelection just behave like this method?

or maybe name this method `effectivelySelectedSubjects`

Reply:
I think making `this.currentSubSelection` behave like this getter is more intuitive.

I like the idea